docs/migration.rst

```diff
@@ -0,0 +1,60 @@
+Migrating from python-fedora
+----------------------------
+
+Fasjson-client provides functionality for the most important endpoints previously exposed in python-fedora.
+Below is a list of common python-fedora endpoints and their fasjson-client alternatives.
+
+
+Creating a client
+******************
+
+The instantiation of a client can be done similarly to python-fedora, except there is no need to provide a username and password with which to authenticate. Instead, this authentication is performed by your service via Kerberos.
+
+For more information, please see :ref:`usage-label`. The content below assumes you have setup your client as detailed in :ref:`usage-label`.
+
+Groups
+******
+
+group_by_id
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You must now use the group `name` instead of `id`. 
+
+    >>> client.get_group(groupname).result
+
+group_by_name
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    >>> client.get_group(groupname, attrs).result
+
+group_members
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    >>> client.get_group_members(groupname, page_size, page_number).result
```

Comment:
It would be nice to give some explanation about pagination, too (or link to the right paragraph)

Reply:
added to the top now. I didn't add `page` to the methods on second glance because i think it might be confusing, instead the link will hopefully explain how to use the response page in your next request.